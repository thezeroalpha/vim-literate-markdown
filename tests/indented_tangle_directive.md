# Indented tangle directive
Having indentation should not matter.

For example, let's say we have stuff in a list:
- Item one
    <!-- :Tangle DIRNAME/FILENAME.out -->
    ```sh
    echo 'this should tangle out.'
    ```
