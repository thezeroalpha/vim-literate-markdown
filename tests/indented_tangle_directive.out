echo 'this should tangle out.'

